native/cocos/bindings/jswrapper/v8/ScriptEngine.h

```diff
@@ -348,6 +348,18 @@ class ScriptEngine final {
     v8::Local<v8::Context> _getContext() const;                   // NOLINT(readability-identifier-naming)
     void _setGarbageCollecting(bool isGarbageCollecting);         // NOLINT(readability-identifier-naming)
 
+    struct DebuggerInfo {
+        ccstd::string serverAddr;
```

Comment:
there's a default value for serverAddr? like 127.0.0.1?

Reply:
I think it doesn't need to be 127.0.0.1 as its default value since we check whether it's valid by the length of serverAddr string.